pybeam/erlang_types.py

```diff
@@ -33,19 +33,26 @@ def index(self):
 
 Port = namedtuple("Port", ["node", "id", "creation"])
 
-Pid = namedtuple("Pid", ["node", "id", "serial", "creation"])
+class String:
+	def __init__(self, value):
+		self.value = value
+	def __eq__(self, other):
+		return self.value == other.value
+	def __repr__(self):
+		return "'%s'" % self.value
+
+class Binary:
+	def __init__(self, value):
+		self._value = value
+	def __eq__(self, other):
+		return self._value == other.value
+	def __repr__(self):
+		return "<<%s>>" % self._value
```

Comment:
Here you revert obsoleted implementation. I think it is better leave `erlang_types` as is, or you may prepare separate PR implementing `__str__` and `__repr__` for `String` and `Binary` types.